packages/next-extraction-plugin/tsconfig.lib.json

```
@@ -4,8 +4,8 @@
     "module": "commonjs",
     "outDir": "../../dist/out-tsc",
     "declaration": true,
-    "types": ["node"]
+    "types": ["node", "environment"]
   },
-  "exclude": ["**/*.spec.ts", "**/*.test.ts"],
-  "include": ["**/*.ts"]
+  "exclude": ["__fixtures__/**/*/*.ts", "**/*.spec.ts", "**/*.test.ts"],
+  "include": ["**/*.ts", "virtual-loader/index.js"]
```

```suggestion
  "exclude": ["**/*.spec.ts", "**/*.test.ts"],
  "include": ["**/*.ts"]
```

We don't have fixtures and `virtual-loader` there, can be this removed?